25/05/13 12:22:27 ERROR GlueExceptionAnalysisListener: [Glue Exception Analysis] {
    "Event": "GlueETLJobExceptionEvent",
    "Timestamp": 1747138947340,
    "Failure Reason": "Traceback (most recent call last):\n  File \"/tmp/Upsert_pyspark_v1.py\", line 2, in <module>\n    import psycopg2\nModuleNotFoundError: No module named 'psycopg2'",
    "Stack Trace": [
        {
            "Declaring Class": "<module>",
            "Method Name": "import psycopg2",
            "File Name": "/tmp/Upsert_pyspark_v1.py",
            "Line Number": 2
        }
    ],
    "Last Executed Line number": 2,
    "script": "Upsert_pyspark_v1.py"
}
